extensions/WindowsSandboxExtension/Providers/WindowsSandboxComputeSystem.cs

```diff
@@ -0,0 +1,190 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.Diagnostics;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Microsoft.Windows.DevHome.SDK;
+using Serilog;
+using Windows.Foundation;
+using Windows.Storage;
+using Windows.Storage.Streams;
+using Windows.Win32;
+using Windows.Win32.Foundation;
+using WindowsSandboxExtension.Helpers;
+using WindowsSandboxExtension.Telemetry;
+
+using Timer = System.Timers.Timer;
+
+namespace WindowsSandboxExtension.Providers;
+
+public class WindowsSandboxComputeSystem : IComputeSystem, IDisposable
+{
+    private readonly ILogger _log = Log.ForContext("SourceContext", nameof(WindowsSandboxProvider));
+    private Process? _windowsSandboxProcess;
+    private ComputeSystemState _state = ComputeSystemState.Stopped;
+
+    private ComputeSystemState State
+    {
+        get => _state;
+
+        set
+        {
+            _state = value;
+            StateChanged?.Invoke(this, value);
+        }
+    }
+
+    public string AssociatedProviderId => Constants.ProviderId;
+
+    public string DisplayName => Resources.GetResource("WindowsSandboxDisplayName", _log);
+
+    public string Id => Guid.NewGuid().ToString();
```

Comment:
Is a new GUID needed for every instance?

Reply:
No, it's not needed. I think maybe in the future we can use the container ID here, but there is no API support for that yet. Should I just return an empty string here?

Reply:
I'd make a private member variable that has `Guid.NewGuid().ToString()` then return that every time